src/fzf.h

```diff
@@ -5,6 +5,12 @@
 #include <stdint.h>
 #include <stddef.h>
 
+#ifdef _WIN32
+    #define DLLEXPORT __declspec(dllexport)
+#else
+    #define DLLEXPORT
+#endif
```

Comment:
Is `__declspec(dllexport)` a vs compiler thing ?

Reply:
It's a Windows thing. By default the visibility of all symbols is hidden (same thing can be achieved on gcc/clang with `-fvisibility=hidden`).